src/form-elements/header-bar.jsx

```diff
@@ -3,16 +3,49 @@
   */
 
 import React from 'react';
-import Grip from '../multi-column/grip';
 import DragHandle from './component-drag-handle';
+import useToggle from '../utils/hooks/useToggle';
 
-export default class HeaderBar extends React.Component {
+const HeaderMenu = () => {
+  const { isOpen, toggle } = useToggle();
+
+  const buttonList = [
+    {
+      icon: 'fas fa-arrows-alt',
+      text: 'Move',
+    },
+    {
+      icon: 'far fa-trash-alt',
```

Comment:
It's a kind of fundamental thing:
If we diced to deeply reworked RFB2, maybe could bring "our" icons? 
The same for buttons, inputs, etc.
Or we still want to add our custom components on our side and just tune some staff in this lib?